rct_optimizations/test/noise_qualification_utest.cpp

```diff
@@ -0,0 +1,126 @@
+#include <rct_optimizations/noise_qualifier.h>
+#include <gtest/gtest.h>
+#include <rct_optimizations_tests/utilities.h>
+#include <rct_optimizations_tests/observation_creator.h>
+
+using namespace rct_optimizations;
+
+TEST(NoiseTest, 2DFullTest)
+{
+  //make target
+  test::Target target(5, 7, 0.025);
+
+  //Load camera intrinsics
+  //camera is a kinect
+  auto camera = test::makeKinectCamera();
+
+  //reserve observations
+  std::size_t obs_cnt = 35;
+  rct_optimizations::Observation2D3D::Set observations;
+  observations.reserve(obs_cnt);
+
+  //create poses outside the loop; would be better to decalre pose, then look at from there
+  Eigen::Isometry3d target_loc = Eigen::Isometry3d::Identity();
+  target_loc.translate(Eigen::Vector3d(1.0,0.0,0.0));
+  Eigen::Isometry3d p = Eigen::Isometry3d::Identity();
+
+  p.translation() = Eigen::Vector3d(0.05, 0, 0.1);
+  p.linear() << 0, 0, 1, -1, 0, 0, 0, -1, 0;
```

Comment:
Please use the `rotate` function instead. This isn't wrong, but I have a hard time looking at these numbers and understanding how you've changed the pose